integration-tests/rita.py

```diff
@@ -213,6 +225,10 @@ def save_rita_settings(id, x):
     toml.dump(x, open("rita-settings-n{}.toml".format(id), "w"))
 
 
+def get_rita_settings(id):
```

Comment:
Come on, not only are you using this only once, but also in a different, self-contained function where this could be inlined. I'd keep this one and delete `get_pubkey`